docs/source/user_guide/basic_usage.rst

```diff
@@ -42,7 +42,7 @@ Receptor can be configured on the command-line, exemplified above, or via a yaml
 
 Start receptor using the config file
 
-.. code::
+.. code-block:: yaml
```

Comment:
Is this YAML? or bash? It looks like a bash command to me.

Reply:
good catch, it is bash